### DISCUSSION TITLE
Update Undici?

### DISCUSSION THREAD

**Discussion by RichGuest** [nodejs/undici#2812] [Q&A]
Hello, I would like to ask how to update Undici on my system? I´m having problem in my project. Simply I cannot download anything through npm to my project. It is showing me that problem is in undici. Info on website is telling me to update to version 6.6.1 to fix it, but I dont know how. 
thanks
![image_2024-02-22_190103294](https://github.com/nodejs/undici/assets/95275892/0baa916d-8527-4930-8407-003fb298af5d)
https://github.com/advisories/GHSA-3787-6prv-h9w3

**Comment by metcoder95** [comment]
Running `npm audit fix` doesn't work?
